src/guides/v2.3/howdoi/admin/customize_admin.md

```diff
@@ -0,0 +1,233 @@
+-  `<Magento_Theme_module_dir>/view/adminhtml/ui_component/design_config_listing.xml`
+
+If you add a certain field as an additional grid column, you also need to set the field’s `use_in_grid property` in the [field’s meta data](#add-fields-metadata) in `di.xml`.
+
+## Customize the design options
+
+These sections detail how to customize form configuration and field metadata.
+
+## Customize the form configuration
+
+The design configuration form is implemented using the form UI component.
+
+To customize the form view, take the following steps:
+
+1. Create an empty `design_config_form.xml` file in the `<your_module_dir>/view/adminhtml/ui_component/` directory.
+
+1. Add content similar to the following:
+
+```xml
+<form xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Ui:etc/ui_configuration.xsd">
+
+
+    <fieldset name="%fieldset_name%">
+        <argument name="data" xsi:type="array">
+            <item name="config" xsi:type="array">
+                <item name="label" xsi:type="string" translate="true">%Fieldset Label as displayed in UI%</item>
+                <item name="sortOrder" xsi:type="number">%order for displaying%</item>
+            </item>
+         </argument>
+        <!--Field sets can be nested -->
+        <fieldset name="%nested_fieldset_name%">
+            <argument name="data" xsi:type="array">
+                <item name="config" xsi:type="array">
+                    <item name="label" xsi:type="string" translate="true">%Nested fieldset Label as displayed in UI%</item>
+                    <item name="collapsible" xsi:type="boolean">true</item>
+                    <!-- Nesting level, the value should correspond to the actual nesting level in the config xml file. For the top field set level = 0 -->
+                    <item name="level" xsi:type="number">%level of nesting%</item>
+                </item>
+            </argument>
+            <field name="%field_name%">
+                <argument name="data" xsi:type="array">
+                    <item name="config" xsi:type="array">
+                        <item name="%field_option1%" xsi:type="%option_type%">%value%</item>
+                        <item name="%field_option2%" xsi:type="%option_type%">%value%</item>
+....
+                    </item>
+                </argument>
+            </field>
+        </fieldset>
+    </fieldset>
+</form>
+```
+
+Your custom fields and field sets will be available for all configuration scopes (website, store, and store view).
+
+Your `design_config_form.xml` is merged with the same files from the other modules. So there is no need to copy their content, you only need to add your customizations.
+
+To customize an existing entity, declare only those options, the values of which are customized, do not copy its entire configuration.
+
+For example, if you only want to rename the **Other Settings** field set, your form configuration must contain the following:
```

Comment:
```suggestion
For example, if you want to rename the **Other Settings** field set only, your form configuration must contain the following:
```

Reply:
I have to disagree with this one. "Only" and "want" - verb with adverb - should go together and not be split this way.

Reply:
Eh, semantics. It's not a big deal and I'm willing to overlook it.